public/src/components/navbar/NavBarCourse.tsx

```diff
@@ -27,6 +27,7 @@ const NavBarCourse = ({ enrollment }: { enrollment: Enrollment }) => {
         }
     }
 
+    const pointerIcon = active ? "icon fa fa-caret-down fa-lg float-right" : " icon fa fa-caret-down fa-rotate-90 fa-lg float-right"
```

Comment:
Please use the multi-line version when long lines like this one. Makes it easier to see the difference between the the two choices.